docs/extend/plugins_volume.md

```diff
@@ -140,15 +162,15 @@ at the first mount request and deprovision at the last corresponding unmount req
 **Response**:
 ```json
 {
-    "Mountpoint": "/path/to/directory/on/host",
+    "Mountpoint": "/path/to/directory/inside/plugin",
```

Comment:
We should be careful to not replace docs for exists in pluginv1 until we deprecate/remove it.

Until then, wherever there's a different in behavior between v1 and v2, there should be 2 paths. In this example, we should retain the `Mountpoint` description for v1 and add an another v2 specific description.